base/log.go

```diff
@@ -11,10 +11,20 @@ import (
 	reporef "github.com/qri-io/qri/repo/ref"
 )
 
+type DatasetLogItem struct {
+	// Decription of a dataset reference
+	dsref.VersionInfo `json:"versionInfo,omitempty"`
+	// Title field from the commit
+	CommitTitle string `json:"commitTitle,omitempty"`
+	// Message field from the commit
+	CommitMessage string `json:"commitMessage,omitempty"`
+}
+
 // DatasetLog fetches the change version history of a dataset
-func DatasetLog(ctx context.Context, r repo.Repo, ref reporef.DatasetRef, limit, offset int, loadDatasets bool) ([]dsref.VersionInfo, error) {
+func DatasetLog(ctx context.Context, r repo.Repo, ref reporef.DatasetRef, limit, offset int, loadDatasets bool) ([]DatasetLogItem, error) {
 	if book := r.Logbook(); book != nil {
 		if versions, err := book.Versions(ctx, reporef.ConvertToDsref(ref), offset, limit); err == nil {
+			items := make([]DatasetLogItem, len(versions))
 			// logs are ok with history not existing. This keeps FSI interaction behaviour consistent
 			// TODO (b5) - we should consider having "empty history" be an ok state, instead of marking as an error
 			if len(versions) == 0 {
```

Comment:
Above the comment a few lines down (starting with "Logbook doesn't store...") to mention the other fields being assigned in this loop.